### DISCUSSION TITLE
Autoregressive Models

### DISCUSSION THREAD

**Discussion by avafor** [MaartenGr/BERTopic#1029] [Q&A]
Hi,
is it in general possible to get the embeddings for BioBERT from autoregressive models such as BioGPT? I think the quality of embeddings are not as good as the encoder models but I would still be curious to try BioGPT since I am working on a biomedical application. 

Thank you and Best Regards,
Avafor

**Comment by MaartenGr** [comment]
As long as the model can be loaded from HuggingFace and can produce embeddings, then it should be no problem using it in BERTopic. You can find more about that [here](https://maartengr.github.io/BERTopic/getting_started/embeddings/embeddings.html).
